fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```

Comment:
Yes, that would be a good place for it to go, and now would be the time to put it in there. I'm ambivalent about it, but if you think that's a good approach, I'll do it.